draft-ietf-httpbis-origin-frame.md

```diff
@@ -153,6 +153,20 @@ The Origin Set is also affected by the 421 (Misdirected Request) response status
 MUST create the ASCII serialisation of the corresponding request's origin (as per {{!RFC6454}},
 Section 6.2) and remove it from the connection's Origin Set, if present.
 
+Note:
+
+: Using an alternative service {{?RFC7838}} can result in using a service that has a different
+  port to the original service (that is, the server that a client contacts when it has no active
+  alternative services).  In that case, the initial origin will have a different port number.  A
+  client could be unable to send requests for the origin that advertised the alternative
+  service.  Explicitly including origins in the ORIGIN frame avoids this problem.
+
```

Comment:
That doesn't ring true either.  I realize that there are cases where the alternative has its own independent identity (e.g., example.com might use alt.example.net as an alternative, but that server might also want to serve resources for alt.example.net), but I wanted to avoid dragging that into this.

The port thing is a trap because the initial origin is primed based on the port that was *used*.  At least with the SNI, the original origin (example.com) is sent, so there is no confusion.  If the port number is the same for origin and alternative, the initial origin covers the resources that caused the client to use this alternative.  Send an empty ORIGIN frame and you still have a usable alternative.  Change the port number and that is no longer true.